build-system-tests/package-lock.json

```diff
@@ -10,6 +10,7 @@
       "devDependencies": {
         "@supercharge/promise-pool": "^2.4.0",
         "commander": "^10.0.1",
+        "esbuild-register": "^3.5.0",
```

Comment:
Not directly related to this change but we added this as a devDep in a previous PR so when you run build tests locally it gets installed